cpp/community_detection_module/algorithm/label_propagation.cpp

```diff
@@ -0,0 +1,291 @@
+#include <cmath>
+#include <mg_graph.hpp>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+#include "label_propagation.hpp"
+
+namespace LabelRankT {
+
+///@brief Checks if set is subset of another set.
+///
+///@tparam T -- set element data type
+///
+///@param set1 -- subset candidate
+///@param set2 -- set against which the check is performed
+///
+///@return -- whether set1 is subset of set2
```

Comment:
From the implementation, it sounds as we are working with vectors, not sets

Reply:
Good catch, thanks!